src/core/components/auth/authorization-popup.jsx

```diff
@@ -16,13 +23,18 @@ export default class AuthorizationPopup extends React.Component {
     return (
       <div className="dialog-ux">
         <div className="backdrop-ux"></div>
-        <div className="modal-ux">
+        <div className="modal-ux"
+          aria-labelledby="modal-ux-title"
+          aria-modal="true"
+          role="dialog"
+          onKeyDown={ this.onKeyDown }>
           <div className="modal-dialog-ux">
             <div className="modal-ux-inner">
               <div className="modal-ux-header">
-                <h3>Available authorizations</h3>
+                <h3 id="modal-ux-title">Available authorizations</h3>
```

Comment:
Using an id here is fine, since the Authorization modal is a singleton within a Swagger UI instance.

**Change request**: We need to note that accessibility might be less than stellar if a user chooses to render Swagger UI instance on a page simultaneously (context: #3372). A good place would be in the README: https://github.com/swagger-api/swagger-ui/blob/master/README.md#known-issues